### PR TITLE
SDK-271 [Java SDK] Update README introduction section

### PR DESCRIPTION
**Pull request by robert-par**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-271](https://incountry.atlassian.net/browse/SDK-271)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version? What is the new version?

### CODE REVIEW

**Review by VladimirLevitskiy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,13 +22,25 @@ For Gradle users please add this line to your dependencies list
...
-----
+## Countries List
 For a full list of supported countries and their codes please [follow this link](countries.md).
 
-Usage
------
-Use `StorageImpl` class to access your data in InCountry using Java SDK.
+
+## Quickstart guide
+To access your data in InCountry Platform by using Java SDK, you need to create an instance of the Storage class using the getInstance method and pass StorageConfig object to it. You can retrieve the `CLIENT_ID`, `CLIENT_SECRET` and `ENV_ID` variables from your dashboard on InCountry Portal.
```
Comment (RommeroInCountry) [inline README.md:30]: Please highlight getInstance, StorageConfig

File: README.md [hunk 2]
```diff
@@ -23,13 +22,25 @@ For Gradle users please add this line to your dependencies list
...
+```java
+
+config = new StorageConfig()
+        .setEnvId(ENV_ID)
+        .setClientId(CLIENT_ID)
+        .setClientSecret(CLIENT_SECRET)
+storageOrdinary = StorageImpl.getInstance(config);
```
Comment (RommeroInCountry) [inline README.md:37]: missing default secret passed to SDK. Please use the simplest one - secret from string

File: README.md [hunk 3]
```diff
@@ -23,13 +22,25 @@ For Gradle users please add this line to your dependencies list
...
+
+```
+
+## Storage Configuration
+
+Below you can find a full list of possible configuration options for creating a Storage instance.
```
Comment (RommeroInCountry) [inline README.md:43]: that's not the full list below. Please provide an extensive javadoc here

File: README.md [hunk 4]
```diff
@@ -75,11 +86,9 @@ API Key authorization is being deprecated. We keep backwards compatibility for `
 
 ---
 
-Parameters `environmentID`, `clientId` and `clientSecret` can be fetched from your dashboard on `Incountry` site.
-
 You can turn off encryption (not recommended) by providing `null` value for parameter `secretKeyAccessor`.
```
Comment (RommeroInCountry) [inline README.md:89]: please remove this line completely (see https://github.com/incountry/sdk-nodejs#storage-configuration)

File: README.md [hunk 5]
```diff
@@ -75,11 +86,9 @@ API Key authorization is being deprecated. We keep backwards compatibility for `
...
 
-Below is an example how to create a storage instance:
+Below you can find API Key authorization usage example:
```
Comment (RommeroInCountry) [inline README.md:91]: this example should be inside of a warning section

**Review by RommeroInCountry — CHANGES_REQUESTED** [review]

**Review by VladimirLevitskiy — DISMISSED** [review]

**Review by VladimirLevitskiy — APPROVED** [review]

**Review by VladimirLevitskiy — DISMISSED** [review]

**Review by VladimirLevitskiy — APPROVED** [review]

**Comment by VladimirLevitskiy** [comment]
New PR will be created instead of force push

**Pull request closed without merging**